### DISCUSSION TITLE
Does PATCH http method invocation invalidate cache?

### DISCUSSION THREAD

**Discussion by ginccc** [resteasy/resteasy#2984] [Q&A] [answered]
The docs state that POST, PUT, and DELETE invalidate cache for that resource. (35.3 - [docs](https://docs.jboss.org/resteasy/docs/5.0.1.Final/userguide/html/Cache_NoCache_CacheControl.html))

```
The cache is also automatically invalidated for a particular URI that has PUT, POST, or DELETE invoked on it. 
```

In my believe PATCH should invalidate the cache the very same way as the other three methods mentioned above. 
Does it?

If not, shouldn't it?

**Comment by jamezp** [comment, marked as answer]
Sorry for the delayed response @ginccc. I don't know the caching well enough, but it looks like it's basically everything but GET and HEAD that get invalidated https://github.com/resteasy/resteasy-extensions/blob/d7b3cb02c5c95ca45d6c7ce509f4988b16d3b211/cache/cache-core/src/main/java/org/jboss/resteasy/plugins/cache/server/ServerCacheHitFilter.java#L47-L54

**Reply by ginccc** [reply]
Perfect. Exactly the code part i was looking for. Works like expected. Thanks @jamezp!
